modules/host/upnp.go

```diff
@@ -99,20 +99,19 @@ func (h *Host) managedForwardPort() error {
 
 	// If the port is invalid, there is no need to perform any of the other
 	// tasks.
-	h.mu.RLock()
-	port := h.port
-	h.mu.RUnlock()
 	portInt, err := strconv.Atoi(port)
 	if err != nil {
 		return err
 	}
 
 	d, err := upnp.Discover()
 	if err != nil {
+		h.log.Printf("WARN: could not automatically forward port %s: no UPnP-enabled devices found: %v", port, err)
```

Comment:
`Discover` already returns an error message that says `"no UPnP-enabled gateway found", so this log might read weird. Just printing the error should be ok